test/helpers/available_languages_helper_test.rb

```diff
@@ -0,0 +1,56 @@
+require 'test_helper'
+
+class AvailableLanguagesHelperTest < ActionView::TestCase
+
+  test "locale_codes extracts the locales from translations" do
+    locales = locale_codes(translations)
+    assert_equal ['en', 'es', 'ar'], locales
+  end
+
+  test "sorted_locales sorts languages by locale with English first" do
+    sorted_translations = sorted_locales(translations)
+    assert_equal ['en', 'ar', 'es'], locale_codes(sorted_translations)
+  end
+
+  test "non_english_translations filters out en locale" do
+    non_english = non_english_translations(translations)
+    assert_equal ['es', 'ar'], locale_codes(non_english)
+  end
+
+  test "native_language_name_for returns native locale names" do
+    locales = ['en', 'es', 'ar'].map {|l| native_language_name_for(l)}
+    assert_equal ["English", "Español", "العربية"], locales
+  end
+
+
+  private
```

Comment:
Our styleguide says that `private` should be outdented: https://github.com/alphagov/styleguides/blob/master/ruby.md#classes